packages/components/src/common/clickable.tsx

```diff
@@ -54,8 +54,8 @@ function Clickable<IComponent extends React.ElementType>({
         color === "brand" && styles.colorBrand,
         color === "neutral" && styles.colorNeutral,
         color === "success" && styles.colorSuccess,
+        color === "warning" && styles.colorWarning,
         // Interactive States (for testing)
-        state === "inactive" && styles.stateInactive,
```

Comment:
intentional deletion?

Reply:
Yes! Sorry, I meant to leave a comment here. I couldn't find any references to `stateInactive` outside of this line.